src/helpers/getBalance.ts

```diff
@@ -3,12 +3,19 @@ import { ethers, providers } from 'ethers'
 const zeroAddress = '0x0000000000000000000000000000000000000000'
 
 export default function (
-  tokenAddress: string,
+  provider: providers.Provider,
   ownerAddress: string,
-  provider: providers.Provider
+  tokenAddress = zeroAddress,
+  tokenId?: number
 ) {
   if (tokenAddress === zeroAddress) {
     return provider.getBalance(ownerAddress)
+  } else if (tokenId) {
```

Comment:
Should we check `tokenId !== undefined` here? Can it be `0`?

Reply:
Year, you right